src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_mysql.py

```diff
@@ -50,7 +50,10 @@ def flexible_server_create(cmd, client,
         cf_availability=cf_mysql_check_resource_availability, cf_private_dns_zone_suffix=cf_mysql_flexible_private_dns_zone_suffix_operations, logging_name='MySQL', command_group='mysql', server_client=client,
         location=location)
 
+    # Process parameters
     server_name = server_name.lower()
+    if high_availability and high_availability.lower() == 'enabled':
+        high_availability = 'ZoneRedundant'
```

Comment:
Why do you need to add `Enabled` if it's only an alternative of `ZoneRedundant`?

Reply:
PMs asked me to make it downward compatible.. so adding back the 'enabled' parameter. What do you think is the best way to deprecate acceptable parameter?

Reply:
Got it. CLI has deprecating mechanism for some argument but no for argument acceptable enums. I guess you can add a warning here to let customer know that `Enabled` is going to be deprecated, use `ZoneRedundant` instead.

Reply:
I see. Let me add that! Thank you :)